ocaml/xcp-rrdd/scripts/rrdd/rrdd-example.py

```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env python
-
-import rrdd, os
-
-if __name__ == "__main__":
```

Comment:
Is the anything that documents the right usage? From the GFS2 component to report, we know it's hard to do it right, so having an example is useful. maybe in the developer docs, like https://xapi-project.github.io/devguide/howtos/metrics.html

Reply:
The link is broken , found an initial commit ,nothing more about the usage.

commit 7522520faa15d8ae98d0530237a61598289d2aca
Author: Rok Strniša <rok.strnisa@citrix.com>
Date: Sat Jun 30 18:00:40 2012 +0100

Added python library rrdd.py, and example plugin rrdd-example.py.

Old XenServer(around 2014) had  rrdd-examples file under   /opt/xensource/libexec/ 

-rwxr-xr-x 1 root root      761 Jul 24 14:59 rrdd-example.py
-rw-r--r-- 1 root root      737 Jul 24 14:59 rrdd-example.pyc
-rw-r--r-- 1 root root      737 Jul 24 14:59 rrdd-example.pyo

Reply:
The link is broken because it's an example of where it should be placed :)